.eslintignore

```diff
@@ -1,3 +1,4 @@
 dist/
 lib/
-node_modules/
\ No newline at end of file
+node_modules/
+__tests__/
```

Comment:
please revert

Reply:
I agree and I will revert it, but I am experience issues when eslint is active for the files in `__test__` directory. 
For example:
![image](https://github.com/user-attachments/assets/35217e06-2c02-42c7-a4dd-36a7c90df34d)

I am using VSCode with the latest version of ESLint. 
Do you happen the know what the issue is?